usaspending_api/spending_explorer/v2/filters/type_filter.py

```diff
@@ -46,8 +46,12 @@ def get_unreported_data_obj(
         result_set.append(condensed_entry)
 
     expected_total = (
-        GTASTotalObligation.objects.filter(fiscal_year=fiscal_year, fiscal_quarter=fiscal_quarter)
-        .values_list("total_obligation", flat=True)
+        GTASTotalObligation.objects.filter(
+            fiscal_year=fiscal_year,
+            fiscal_period__gt=((fiscal_quarter - 1) * 3),
+            fiscal_period__lte=(fiscal_quarter + 1) * 3,
```

Comment:
For some reason I'm not following these changes.

If the fiscal quarter is 1, isn't this now checking for `fiscal_period` > 0 and `fiscal_period` <= 6?
For quarter 4, it would be checking `fiscal_period` > 9 and `fiscal_period` <= 15